include/IECore/TransformationMatrix.inl

```diff
@@ -81,6 +81,21 @@ std::ostream &operator << ( std::ostream &os, const TransformationMatrix<T> &x )
 	return os;
 }
 
+template<class T>
+void murmurHashAppend( IECore::MurmurHash &h, const TransformationMatrix<T> &data )
+{
+	h.append( data.scalePivot );
+	h.append( data.scalePivot );
+	h.append( data.scale );
+	h.append( data.shear );
+	h.append( data.scalePivotTranslation );
+	h.append( data.rotatePivot );
+	h.append( data.rotationOrientation );
+	h.append( data.rotate );
```

Comment:
We're missing `rotate.order` here, since rotate is an `Euler` and we don't seem to have `murmurHashAppend( const Euler & )` (I presume we're getting the version for the `Vector3` base instead?). We should probably add the Euler version and then this code can stay the same.

Reply:
Now we have this, we should remove the hash specialisation from `TransformationMatrixData.cpp`.

Reply:
Fixed squashed into 467a02b